client/components/Navbar.js

```diff
@@ -5,29 +5,33 @@ import {logout} from '../store'
 
 const Navbar = ({handleClick, isLoggedIn, isAdmin}) => {
   return (
-  <div>
-    <h1>FS-App-Template</h1>
-    <nav>
-      {isLoggedIn ? (
-        <div>
-          {/* The navbar will show these links after you log in */}
-          <Link to="/home">Home</Link>
-          {isAdmin ? <Link to="/admin-tools">Admin Tools</Link> : ''}
+    <div>
+      <h1>Magical Merchants</h1>
+      <nav>
+        <Link to="/cart">
+          <img src="white-cart.svg" className="cartIcon" alt="Cart"></img>
+        </Link>
+
+        {/* These links will display if there is NOT a user logged in. */}
+        {!isLoggedIn && <Link to="/login">Login</Link>}
```

Comment:
For the routes these need to be broken into single lines, specifically because React Router isn't able to find Route components nested in other components. But here you could group them under one conditional if you wanted (you'd just have to place them inside of a React.Fragment, which is basically an empty jsx tag)

No need to change though, just for your own info